.github/workflows/stale.yml

```
@@ -1,54 +1,56 @@
           stale-pr-message: >
             This PR has had no activity for 14 days.
             It will be closed in 7 days unless there is new activity.
-
-          # Message posted when a stale PR is closed
           close-pr-message: >
             Closing due to inactivity. Re-open when ready to continue.
 
-          # Number of days without activity before marking as stale
-          days-before-stale: 14
+          # --- Discussions ---
+          stale-discussion-message: >
```

actionlint (1.7.12)

[error] input "stale-discussion-message" is not defined in action "actions/stale@v9". available inputs are "any-of-issue-labels", "any-of-labels", "any-of-pr-labels", "ascending", "close-issue-label", "close-issue-message", "close-issue-reason", "close-pr-label", "close-pr-message", "days-before-close", "days-before-issue-close", "days-before-issue-stale", "days-before-pr-close", "days-before-pr-stale", "days-before-stale", "debug-only", "delete-branch", "enable-statistics", "exempt-all-assignees", "exempt-all-issue-assignees", "exempt-all-issue-milestones", "exempt-all-milestones", "exempt-all-pr-assignees", "exempt-all-pr-milestones", "exempt-assignees", "exempt-draft-pr", "exempt-issue-assignees", "exempt-issue-labels", "exempt-issue-milestones", "exempt-milestones", "exempt-pr-assignees", "exempt-pr-labels", "exempt-pr-milestones", "ignore-issue-updates", "ignore-pr-updates", "ignore-updates", "include-only-assigned", "labels-to-add-when-unstale", "labels-to-remove-when-stale", "labels-to-remove-when-unstale", "only-issue-labels", "only-labels", "only-pr-labels", "operations-per-run", "remove-issue-stale-when-updated", "remove-pr-stale-when-updated", "remove-stale-when-updated", "repo-token", "stale-issue-label", "stale-issue-message", "stale-pr-label", "stale-pr-message", "start-date"

(action)